CMakeLists.txt

```diff
@@ -136,6 +136,7 @@ elseif (TARGET_PLATFORM STREQUAL "rdkb")
     # Disable file and enable syslog logging
     set(BEEROCKS_LOG_FILES_ENABLED "false")
     set(BEEROCKS_LOG_SYSLOG_ENABLED "true")
+    set(BEEROCKS_LOG_FILES_AUTO_ROLL "false")
```

Comment:
I think I'm missing something here... If log files are disabled and instead we're logging to syslog, what is auto roll still doing? Actually the LOG_FILES_PATH and LOG_FILES_SUFFIX changes have the same issue.

Reply:
The beerocks log performs a different roll then the Syslog.
The syslog roll done by writing to the beginning of the file. If you ask me it is less convenient.
Beerocks roll put all the logs in an archive and create new logs files. At most, only the latest 3 archives are saved.